blameTheISP.py

```diff
@@ -47,7 +48,11 @@ def toggle_ping_only():
 
 def main():
 
-    speed_test_opts = ConfigJson('~/Blame-The-ISP/blameTheISP.config.json')
+    # Probably a better way to do this, but I think this is cron safe.
+    speedtest_dir = os.path.realpath(sys.argv[0])
+    opts_path = '/'.join(speedtest_dir.split('/')[:-1])
+    opts_path += '/blameTheISP.config.json'
```

Comment:
Might I suggest replacing lines 52-54 with:
`opts_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'blameTheISP.config.json')`
OR
`speedtest_dir = os.path.abspath(os.path.dirname(__file__))`
`opts_path = os.path.join(speedtest_dir, 'blameTheISP.config.json'`

These methods do not assume OS path notations ('/' vs '\'), don't require the import of sys, and are proven to be cron-safe.